rl_algorithms/dqn/agent.py

```diff
@@ -126,31 +132,17 @@ def _initialize(self):
     def _init_network(self):
         """Initialize networks and optimizers."""
 
-        if self.use_conv:
-            # create CNN
-            self.dqn = dqn_utils.get_cnn_model(
-                self.hyper_params, self.action_dim, self.state_dim, self.network_cfg
-            )
-            self.dqn_target = dqn_utils.get_cnn_model(
-                self.hyper_params, self.action_dim, self.state_dim, self.network_cfg
-            )
+        self.head_cfg.params["input_size"] = calculate_fc_input_size(
```

Comment:
should modify from `self.head_cfg.params["input_size"]` to `self.head_cfg.params.input_size` because we use `ConfigDict` class.

Reply:
should modify others configs

Reply:
reflect the issue in commit 1070159.